tests/confirmation_test.js

```diff
@@ -0,0 +1,18 @@
+Feature('confirmation');
+
+Scenario('Verifcation of confirmation events', async ({ I }) => {
+    I.amOnPage('/');
+    let confirmations = await I.executeScript(async () => {
+        let confirmationsArray = [];
+        evolv.context.remoteContext.confirmations.forEach(element => {
+            console.log(element.cid)
+            confirmationsArray.push(element.cid);
+        })
+        return confirmationsArray;
+       
+    });
+    //prints cid of confirmation
+    console.log(confirmations)
+    //checking that confirmations array is not empty
+    I.assertNotEqual(confirmations.length, 0);
```

Comment:
Nitpick: the comments are not necessary. The code should speak for itself. Comments that explain intent can quickly get out of sync when code changes and then can lead to confusion later on when the comment and code have a discrepancy.

Use explanatory comments sparingly in the cases where you can't capture the intent in the code itself.
```suggestion

    console.log(confirmations)

    I.assertNotEqual(confirmations.length, 0);
```

Reply:
fixed